Contacts/People/Council/council.md

```diff
@@ -54,25 +53,27 @@ The present ({{site.data.release.date | date: "%Y" }}) members of the GAP Counci
 - Max Neunhöffer (since August 2011)
 - Alice Niemeyer (since August 2011)
 - Götz Pfeiffer (since September 2007)
-- Edmund Robertson (since June 1995)
 - Leonard Soicher (since June 1997)
 
-Chair: Leonard Soicher (since March 2009)
+Chair: Bettina Eick (since October 2021)
 
 Former members of the GAP Council are:
 
 - Mike Atkinson (from June 1995 to June 2001)
 - Andrea Caranti (from June 1995 to June 2001)
+- Gene Cooperman (from June 1995 to October 2021)
 - Mike F. Newman (from June 1995 to November 2011)
-- Joachim Neubueser (from June 1999 to March 2006)
+- Joachim Neubüser (from June 1999 to March 2006)
 - Eamonn O'Brien (from June 1997 to February 2003)
 - Herbert Pahlings (from June 1995 to September 2007)
 - Cheryl Praeger (from June 1995 to August 2016)
 - Sarah Rees (from August 1999 to June 2001)
+- Edmund Robertson (from June 1995 to October 2021)
```

Comment:
I think @Stefan-Kohl means that a range date e.g. "from June 1995 to October 2021" is typically inclusive in English, so for example, this time period includes all of October 2021. Is that what you intend?

Reply:
OK, I see, indeed! Thanks for pointing it out, I've fixed it now.